### ISSUE TITLE
CodeFixes mishandle qualified receivers: PC0030 drops 'this.' qualifier; MandatoryFieldMissingOnApiPage misses this-qualified/quoted receivers

### ISSUE THREAD

**Issue by Arthurvdv** [ALCops/Analyzers#440]
### Analyzer

PlatformCop

### Issue type

Other

### Rule ID

PC0030 (UsePartialRecordsOnRead), MandatoryFieldMissingOnApiPage CodeFix

### Description

Follow-up to #428 (PC0035 CodeFix generated the quoted identifier `"this.Job"` for `this.`-qualified receivers). An audit of all CodeFixes for the same instance-expression string handling found two more spots that mishandle qualified receivers. Both are lower severity than #428 (no non-compiling code is generated), but they should be aligned with the pattern used in the #428 fix: reuse the original receiver syntax node (or compare via symbols) instead of round-tripping through strings.

**1. PC0030 `UsePartialRecordsOnRead.cs` (`BuildSetLoadFieldsStatement`)**

The CodeFix builds the `SetLoadFields` receiver from `variableSymbol.Name`, silently dropping the `this.` qualifier:

```al
var
    Job: Record Job;

    this.Job.Get(JobNo);
```

produces

```al
Job.SetLoadFields(...);   // expected: this.Job.SetLoadFields(...);
this.Job.Get(JobNo);
```

This compiles, but:
- If a local variable or parameter shadows the global `Job`, the generated statement targets the wrong variable (behavior change).
- The output is style-inconsistent with the surrounding code.

Fix direction: pass the original receiver expression node (`memberAccess.Expression.WithoutTrivia().WithLeadingTrivia(SyntaxFactory.ElasticMarker)`) into the statement builder, as done for PC0035 in the fix for #428. The elastic marker is required so the CodeAction formatter indents the inserted statement.

**2. `MandatoryFieldMissingOnApiPage.cs:99` (receiver string comparison)**

```csharp
SemanticFacts.IsSameName(f.Expression.ToString(), $"Rec.{mandatoryField.RecField}")
```

This textual comparison misses equivalent receiver spellings:
- `this.Rec.SystemId` (`this.`-qualified)
- `Rec."SystemId"` / quoted identifiers

resulting in an existing field control not being recognized, so the diagnostic fires incorrectly and the CodeFix can insert a duplicate field control.

Fix direction: compare via symbols (`GetSymbolSafe()`) or compare the member name and receiver symbol separately instead of comparing rendered syntax strings.

**Verified safe (no action needed):** LC0083, FC0005, LC0004 (RecordInstanceIsolationLevel), PC0037 and the remaining CodeFixes either reuse the original syntax nodes or build fixed method names only.

### AL code to reproduce

```al
var
    Job: Record Job;

procedure ReadJob(JobNo: Code[20])
begin
    this.Job.Get(JobNo);
end;
```

### Additional context

Pattern to follow: PR for #428 (PC0035 `UseSetAutoCalcFieldsForLoopsCodeFixProvider`).
